src/components/AppComponent.js

```diff
@@ -41,9 +41,24 @@ const fontSizeLocal = +(localStorage['Settings/Font Size'] || 16)
 const tutorialLocal = localStorage['Settings/Tutorial'] === 'On'
 const tutorialStepLocal = +(localStorage['Settings/Tutorial Step'] || 1)
 
+/** A container that scales its children by the given amount. */
+const Scale = ({ amount, children }) =>
```

Comment:
Since this component is independent of AppComponent, maybe we can move it out

Reply:
Agree, thanks